smooth_ctrl: 
  ros__parameters:
    k1: 1.0
    k2: 3.0
    beta: 0.4
    lambda: 2.0
    v_max: 0.5
    home_pose:
      x: 0.0
      y: 0.0
      theta: 0.0
    active: false
    pose_sub_topic: amcl_pose
    goal_sub_topic: move_base_simple/goal
    cmd_vel_pub_topic: cmd_vel_mux/input/teleop
